config/crd/bases/velero.io_backupstoragelocations.yaml

```diff
@@ -87,6 +87,24 @@ spec:
                 type: string
               description: Config is for provider-specific configuration fields.
               type: object
+            credential:
+              description: Credential contains the credential information intended
+                to be used with this location
+              properties:
+                key:
+                  description: The key of the secret to select from.  Must be a valid
+                    secret key.
+                  type: string
+                name:
+                  description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                    TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
Is this TODO for the description field? I think the link should be enough, IMO.

Reply:
This is pulled in directly from the `LocalObjectReference` type. I don't think it's something that we can control?

Reply:
Ah, you're right, I see that now. Ok, we'll live with it :)